bloom-housing/bloom
# feat: application get endpoints

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

The application get endpoints in the prisma rewrite along with all of the models needed for that and proper seeding of the data

This also has the upgrade to prisma 5.0.0 which required a couple of mostly minor changes but did require the upgrade of Typescript.

## How Can This Be Tested/Reviewed?

Locally pulled down you should be able to run the seed commands and get data populated into the applications and applicant tables. And you should be able to see the applications get endpoints here http://localhost:3101/api

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
